README.md

```diff
@@ -1,7 +1,300 @@
+- For searches with unspecific crosslinkers like SDA, include prefilters (or boost) on minimum number of fragments. Include also a prefilter on "fragment unique crosslinked matched conservative" and exclude consecutives. This ensures that only spectral matches with at least one fragment proving a crosslinked peptide pair are considered.
+
+#### Watch out for:
+- The program warns if there aren't sufficient targets to estimate FDR accurately. If this warning refers to the level of analysis you are interested in, there is likely almost nothing in the data. Unless prefilters were set way too stringent.
+- The FDR calculation rests on the assumption that target-decoy pairs have twice the chance of random matching than decoy-decoy pairs (TD+DT vs DD). If in your results you have more decoy-decoy (DD) than target-decoy (TD), your FDR evaluates to a negative number and becomes meaningless. This may be a sign that there are no crosslinks in the datasets, or that prefilters are not working as intended.
+
+
+### Results summary
+After the calculation is complete (check the log tab, the lower bottom left of the window or the "calculate" button becoming clickable again), the results tab presents a summary of all matches passing the FDR threshold with the number of decoys and target-decoys in each category. Remember the ratio of targets and decoys passing determines the accuracy of the FDR estimation.
+
+### Writing out search results
+
+Results may be written out in xiFDR .csv format (csv tab) or in mzIdentML1.2.0 format. We advise to do both every time, as the summary file is useful to keep track of what was done, and the mzIdentML for later deposition or upload to [xiView.org](https://www.xiview.org).
+
+#### csv output 
+will generate several files:
+
+- Summary file: a file containing a summary of the FDR calculation, results and all settings
+- CSM file: a file containing all crosslink spectra matches passing the threshold. This file can be uploaded to xiview.org for visualization
+- peptide pairs file: a file containing all peptide pairs passing the threshold
+- protein groups file: all protein groups passing the threshold (i.e. including ambiguity when a peptide cannot be assigned to an individual protein)
+- Links file: all the crosslinked residue pairs passing the threshold
+- PPI file: all the protein-protein interactions passing the thresold.
+
+Each file contains information about the FDR, local FDR (if enabled), posterior error probability, target/decoy nature of each match, as well as the peakfile of origin.
+
+Beware that if multiple FDR thresholds are set, *only* the matches passing the highest level of aggregation are included in the results. In other words, the csv files for a search with both a 5% residue pair FDR and 5% protein pair FDR will only include residue pairs with a 5% or better FDR that also lead to a protein pair with a 5% or better FDR.
```

Comment:
```suggestion
Beware that if multiple FDR thresholds are set, *only* the matches passing all levels of aggregation are included in the results. In other words, the csv files for a search with both a 5% residue pair FDR and 5% protein pair FDR will only include residue pairs with a 5% or better FDR that also lead to a protein pair with a 5% or better FDR.

In effect will see (potentially a lot) less decoys for a FDR level then the FDR indicates. E.g. when filtering for both 5% residue pair FDR and 5% protein pair FDR the resulting residue-pair file will probably contain very few remaining decoys. *BUT* you can't take that as an indication, that you have next to no false positives left as the applied PPI filter basically invalidates any follow up decoy based error estimation on residue pairs. In general any higher level of abstraction FDR will make any follow up observation of decoys on lower levels meaningless. This only applies for going from higher levels to lower levels (e.g. PPI->residue pair->peptide pair->CSM). The other way around (CSM->peptide pair->residue pair->PPI) is a valid  process and is how xiFDR internally works.
```

Reply:
i think the english here is a bit confusing

Reply:
My English not perfect is? Impossible!

Will try to improve - but do you think it make sense to extend that warning like that?

Reply:
"In effect will see (potentially a lot) less decoys for a FDR level then the FDR indicates. E.g. when filtering for both 5% residue pair FDR and 5% protein pair FDR the resulting residue-pair file will probably contain very few remaining decoys. *BUT* you can't take that as an indication, that you have next to no false positives left as the applied PPI filter basically invalidates any follow up decoy based error estimation on residue pairs. In general any higher level of abstraction FDR will make any follow up observation of decoys on lower levels meaningless. This only applies for going from higher levels to lower levels (e.g. PPI->residue pair->peptide pair->CSM). The other way around (CSM->peptide pair->residue pair->PPI) is a valid  process and is how xiFDR internally works."

I would change to

"This can result in a lower number of decoys at a given FDR level than the reported FDR number. Meaning, when when filtering for both 5% residue pair FDR and 5% protein pair FDR the resulting residue pair file will contain only decoys that reach the protein pair level. This should not be misconstrued as having next to no false positives left at the residue pair level: the protein pair filter basically invalidates any follow up decoy based error estimation at the lower residue pair level. Thus, any FDR at a higher level of aggregation (eg. protein pair vs CSM) will prevent an analysis of decoys at lower levels, as error is propagated through."

Reply:
yes i think informing the user in more detail here is good. I don't even think many people in juri's lab know this one

Reply:
```suggestion
Beware that if multiple FDR thresholds are set, *only* the matches passing all levels of aggregation are included in the results. In other words, the csv files for a search with both a 5% residue pair FDR and 5% protein pair FDR will only include residue pairs with a 5% or better FDR that also lead to a protein pair with a 5% or better FDR.

This can result in a lower number of decoys at a given FDR level than the reported FDR number. Meaning, when when filtering for both 5% residue pair FDR and 5% protein pair FDR the resulting residue pair file will contain only decoys that reach the protein pair level. This should not be misconstrued as having next to no false positives left at the residue pair level: the protein pair filter basically invalidates any follow up decoy based error estimation at the lower residue pair level. Thus, any FDR at a higher level of aggregation (eg. protein pair vs CSM) will prevent a meaningful analysis of false positives via decoys at lower levels.
```